Sources/PackageDescription4/SupportedPlatforms.swift

```diff
@@ -0,0 +1,110 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2018 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+public struct SupportedPlatform: Encodable {
+
+    let platform: String
+    let version: VersionedValue<String>?
+
+    init(platform: String, version: VersionedValue<String>? = nil) {
+        self.platform = platform
+        self.version = version
+    }
+
+    public static func macOS(_ version: SupportedPlatform.MacOSVersion) -> SupportedPlatform {
+        return SupportedPlatform(platform: "macosx", version: version.version)
```

Comment:
Where are these values used? If they are internal only, there's no need to use the SDK names as the identifiers; just use the modern values that correspond to the ones we pass in the target triple: macos, ios, tvos, and watchos.

Reply:
sure